packages/aws-otel-collector/0001-logger-change-collector-log-file-path.patch

```diff
@@ -0,0 +1,26 @@
+From d5897e078f85c9a41ecce024fc4a409eebfa0bda Mon Sep 17 00:00:00 2001
+From: Kush Upadhyay <kushupad@amazon.com>
+Date: Wed, 24 Jul 2024 06:21:21 +0000
+Subject: [PATCH] logger: change collector log file path
+
+Signed-off-by: Kush Upadhyay <kushupad@amazon.com>
+---
+ pkg/logger/logger.go | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/pkg/logger/logger.go b/pkg/logger/logger.go
+index 9805eeab..7fcbab31 100644
+--- a/pkg/logger/logger.go
++++ b/pkg/logger/logger.go
+@@ -31,7 +31,7 @@ import (
+ )
+ 
+ var (
+-	UnixLogPath      = "/opt/aws/aws-otel-collector/logs/aws-otel-collector.log"
++	UnixLogPath      = "/var/log/aws/aws-otel-collector.log"
```

Comment:
We should be extra safe and patch this file as well

https://github.com/aws-observability/aws-otel-collector/blob/311c612da46c22aa0e32d3787dd422bbaba8e83d/pkg/extraconfig/extraconfig.go#L26

We should encourage dowstreams to not read configuration files from `/opt`, and instead use standard configuration directories, like `/etc`.

Reply:
Got it, I'll change the current `/opt/aws/aws-otel-collector/etc/extracfg.txt` path to `/etc/extracfg.txt`